joomla/joomla-cms
# [4.0] Don't ship .es6 files

Closes https://github.com/joomla/joomla-cms/issues/29092.

### Summary of Changes

Excludes .es6 files from being copied to media directory.

### Testing Instructions

Download and install this package https://ci.joomla.org/artifacts/joomla/joomla-cms/4.0-dev/31201/downloads/36975/Joomla_4.0.0-beta5-dev+pr.31201-Development-Full_Package.zip.
Check that JS functionality still works (e.g. messages, modals, form validation).
Check if `media/system/js/core.es6` directory exists.

### Expected result

`media/system/js/core.es6` directory doesn't exist and JS functionality works properly.

### Documentation Changes Required

No.
